needle/modules/storage/data/files_plist.py

```diff
@@ -8,11 +8,16 @@ class Module(BaseModule):
         'name': 'Plist Files',
         'author': '@LanciniMarco (@MWRLabs)',
         'description': 'List plist files contained in the app folders, alongside with their Data Protection Class. '
-                       'Plus, offers the chance to inspect them with Plutil',
+                       'Plus, offers the chance to inspect them with Plutil. '
+                       'Alternatively, you can convert and dump all of the plist files for local analysis.',
         'options': (
-            ('analyze', True, True, 'Prompt to pick one file to analyze'),
-            ('output', True, False, 'Full path of the output file')
+            ('analyze', True, True, 'Prompt to pick one file to analyze (ignored if "DUMP_ALL" is True)'),

```

Comment:
I wouldn't make it exclusive. You could analyze 1 file AND dump everything anyway afterwards